packages/docs/src/pages/tutorial/store/recursive/index.mdx

```diff
@@ -3,11 +3,16 @@ title: Recursive Store
 layout: tutorial
 ---
 
-In this example, we have created a store that contains other objects rather than just primitives. (`store.counter` is object literal.)
+This example features a store that contains other objects rather than just primitives. The `store.counter` property is an object literal.
 
-The example does not work, because by default the `useStore` only sets up watches on top-level properties. To make the example work, we need to either:
-1. wrap the `{count:0}` in another `useStore()` call, which is cumbersome OR
-2. ask the `useStore` to apply reactivity to all properties recursively by passing the additional argument `{recursive:true}`.
+The example doesn't work as is, because by default because the `useStore` function only watches top-level properties.
```

Comment:
You have a typo here (inception anyone? 😅) - double "because"

Reply:
🤦 - grateful for the second pair of eyes!